Cart Flow.xcodeproj/project.xcworkspace/xcuserdata/scottquintana.xcuserdatad/IDEFindNavigatorScopes.plist

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Typically files in `*xcuserdatad` are not committed to git repositories, as they only apply to you and not others working on the project. You can ignore files in git by adding a `.gitignore` file to the root directory of your project. Here's a typical `.gitignore` file I start with on Swift projects: https://github.com/github/gitignore/blob/master/Swift.gitignore.